ca/config.go

```diff
@@ -214,6 +189,11 @@ func (s *SecurityConfig) IssuerInfo() *IssuerInfo {
 	return s.issuerInfo
 }
 
+// Close closes the security config, freeing up the watch queue resource
+func (s *SecurityConfig) Close() {
```

Comment:
We mainly create the security config in the `node/node.go` code, and the watch gets cleaned up at the end of `run` anyway, so holding onto the cancel seems fine?  Returning the cancel may make it harder to leak in tests, though, so:  https://github.com/docker/swarmkit/pull/2274